compose/install.md

```diff
@@ -34,28 +34,59 @@ To install Compose, do the following:
     Invoke-WebRequest "https://github.com/docker/compose/releases/download/$dockerComposeVersion/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
     ```
 
-    For example, to download Compose version 1.12.0, the command is:
+    For example, to download Compose version 1.13.0, the command is:
 
     ```none
-    Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.12.0/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
-        ```
+    Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.13.0/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
+    ```
+    >**Use the latest Compose release number in the download command.**
+    As already mentioned, the above command is an _example_, and it may become
+out-of-date once in a while. Always follow the command pattern shown above it.
```

Comment:
Per our discussion (summarized below), I'm leaving these as is, but I did them the way you like on https://docs.docker.com/test/#admonitions-notes. 

Pros and cons to both approaches:
- On the one hand, if you line them up and use `>` at start of every line, it is aesthetically satisfying.

- On the other hand, you have a lot of `>` all over the place, and if you want to do an auto line wrap on atom for uninterrupted line breaks, you have to take all those carrots out and put them back in later.